resources/views/mail/dues/transactionreminder.blade.php

```diff
@@ -0,0 +1,21 @@
+Hi {{ $user->preferred_first_name }},
+
+@if(\App\Models\DuesPackage::userCanPurchase($user)->count() > 1)
+You still need to pay dues for this semester. Please visit {{ route('home') }} to begin the process.
+
+@php
+print(implode(" and ", \App\Models\DuesPackage::userCanPurchase($user)->orderByDesc('cost')->get()->map(static fn (\App\Models\DuesPackage $package, int $key): string => $package->name.' dues are $'.intval($package->cost) )->toArray()))
```

Comment:
What happens if there are more than 2 available dues packages? Why not just limit to the top 2?

Reply:
If there are more than two then you end up with a slightly weirdly phrased list. It still works fine.

In practice there should only be 0, 1, or 2 packages available for any individual person to purchase, so I don't think it should be an issue.